deploy/deploy_file_server.yml

```diff
@@ -1,12 +1,16 @@
 # deploy_file_server.yml
 ---
 - hosts: file_server
-  sudo: yes
+  remote_user: vagrant
```

Comment:
Please remove the "remote_user" line, as this isn't needed and will confuse Ansible when deploying to non-vagrant environments.